Content/Tiles/CrashTech/CrashPod.cs

```diff
@@ -1,91 +1,132 @@
+                WorldGen.KillTile(ParentX, ParentY);
+                NetMessage.SendTileSquare(Player.whoAmI, (int)(Projectile.position.X / 16f), (int)(Projectile.position.Y / 16f), 2, 3, TileChangeType.None);
 
-                        Framing.GetTileSafely(realX, realY).TileFrameX += 36;
-                    }
+                Core.Systems.CameraSystem.Shake += 4;
+
+                for (int k = 1; k <= 5; k++)
+                    Gore.NewGoreDirect(Projectile.GetSource_Death(), Projectile.position + new Vector2(Main.rand.Next(Projectile.width), Main.rand.Next(Projectile.height)), (Vector2.Normalize(Player.velocity) * Main.rand.NextFloat(4)).RotatedByRandom(MathHelper.ToRadians(35f)), Mod.Find<ModGore>("CrashPodGore" + k).Type);
+
+                for (int i = 0; i < 17; i++)
+                {
+                    //for some reason the BuzzSpark dust spawns super offset 
+                    Dust.NewDustPerfect(Projectile.Center + new Vector2(0f, 28f) + Main.rand.NextVector2Circular(15,25), ModContent.DustType<Dusts.BuzzSpark>(), (Vector2.Normalize(Player.velocity) * Main.rand.NextFloat(9.5f)).RotatedByRandom(MathHelper.ToRadians(5f)), 0, new Color(255, 255, 60) * 0.8f, 1.15f);
 
-                Loot[] smallLoot = new Loot[5];
+                    Dust.NewDustPerfect(Projectile.Center + Main.rand.NextVector2Circular(15, 25), ModContent.DustType<Dusts.Glow>(), (Vector2.Normalize(Player.velocity) * Main.rand.NextFloat(9)).RotatedByRandom(MathHelper.ToRadians(15f)), 0, new Color(150, 80, 40), Main.rand.NextFloat(0.25f, 0.5f));
+                }
 
-                List<Loot> types = Helper.RandomizeList(SmallLootPool);
-                for (int k = 0; k < 5; k++) smallLoot[k] = types[k];
 
-                UILoader.GetUIState<LootUI>().SetItems(GoldLootPool[Main.rand.Next(GoldLootPool.Count)], smallLoot);
-                UILoader.GetUIState<LootUI>().Visible = true;
-                return true;
+                Terraria.Audio.SoundEngine.PlaySound(SoundID.Shatter, Projectile.Center);
             }
-            return false;
+        }
+    }
+    public class CrashPodGTile : GlobalTile
+    {
+        public override bool CanExplode(int i, int j, int type)
+        {
+            if (Main.tile[i, j - 1].TileType == ModContent.TileType<CrashPod>())
+                return false;
+            return base.CanExplode(i, j, type);
         }
 
-        internal override List<Loot> GoldLootPool =>
-            new List<Loot>
-            {
-                new Loot(ItemID.DirtBlock, 1)
-            };
+        public override bool CanKillTile(int i, int j, int type, ref bool blockDamaged)
+        {
+            if (Main.tile[i, j - 1].TileType == ModContent.TileType<CrashPod>())
+                return false;
+            return base.CanKillTile(i, j, type, ref blockDamaged);
+        }
 
-        internal override List<Loot> SmallLootPool =>
-            new List<Loot>
+        public override void RandomUpdate(int i, int j, int type)
```

Comment:
treating these like grass tiles seems a bit strange? It would seem to work but still feels a bit strange in principle.